web/app_dev.php

```diff
@@ -7,14 +7,16 @@
 // read http://symfony.com/doc/current/book/installation.html#configuration-and-setup for more information
 //umask(0000);
 
-// This check prevents access to debug front controllers that are deployed by accident to production servers.
-// Feel free to remove this, extend it, or make something more sophisticated.
-if (isset($_SERVER['HTTP_CLIENT_IP'])
-    || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
-    || !(in_array(@$_SERVER['REMOTE_ADDR'], array('127.0.0.1', 'fe80::1', '::1')) || php_sapi_name() === 'cli-server')
-) {
-    header('HTTP/1.0 403 Forbidden');
-    exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
+if (strtolower($_SERVER['HTTP_HOST']) != 'pipelinedashboard.local') {
```

Comment:
Have you had a poke here
http://symfony.com/doc/current/cookbook/configuration/web_server_configuration.html

On Mon, Mar 9, 2015 at 4:31 PM, Chris Moreton <notifications@github.com>
wrote:

> In web/app_dev.php
> <https://github.com/DashboardHub/PipelineDashboard/pull/67#discussion_r26051047>
> :
>
> > @@ -7,14 +7,16 @@
> >  // read http://symfony.com/doc/current/book/installation.html#configuration-and-setup for more information
> >  //umask(0000);
> >
> > -// This check prevents access to debug front controllers that are deployed by accident to production servers.
> > -// Feel free to remove this, extend it, or make something more sophisticated.
> > -if (isset($_SERVER['HTTP_CLIENT_IP'])
> > -    || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
> > -    || !(in_array(@$_SERVER['REMOTE_ADDR'], array('127.0.0.1', 'fe80::1', '::1')) || php_sapi_name() === 'cli-server')
> > -) {
> > -    header('HTTP/1.0 403 Forbidden');
> > -    exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
> > +if (strtolower($_SERVER['HTTP_HOST']) != 'pipelinedashboard.local') {
>
> Which var?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DashboardHub/PipelineDashboard/pull/67/files#r26051047>
> .
>